scripts/apothecary/apothecary

```diff
@@ -88,7 +88,8 @@ VS_64_BIT_ENV='VC\bin\x86_amd64\vcvarsx86_amd64.bat'
 # paths to android SDK, etc
```

Comment:
np, also the travis config in that repo aready auto builds the latest
libraries but you'll need to change it to download v12 of the NDK i guess

On 22.06.2016 22:08, Jonas Jongejan wrote:
> In scripts/apothecary/apothecary
> <https://github.com/openframeworks/openFrameworks/pull/5059#discussion_r68125322>:
> 
>> @@ -88,7 +88,8 @@ VS_64_BIT_ENV='VC\bin\x86_amd64\vcvarsx86_amd64.bat'
>>  # paths to android SDK, etc
> 
> Ohhh, thanks for noticing it!
> 
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openframeworks/openFrameworks/pull/5059/files/23dd4480d0d93430a6dc9e3591df3906293ecc71..6f53cd61fb363bf6daa079d7d992d2ea6fec6ec0#r68125322>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAC8cAl4JfV_qa19J6X5CP2yIRZI06jkks5qOZZKgaJpZM4II_Tu>.
>